frontend/app/src/theme/community/tokyonight.ts

```diff
@@ -0,0 +1,77 @@
+import type { Theme } from "../types";
+
+// Colors
+const white = "#FFFFFF";
+const offWhite = "#DFDFDF";
+const black = "#000000";
+const raspberry = "#971E57";
+const darkGray = "#696969";
+const silver = "#C0C0C0";
+const veryDarkGray = "#1e2123";
+const lighterDarkGray = "#101010";
+const txtLight = offWhite;
+const notificationAccent = silver;
+
+export function getTheme(base: Theme): Theme {
+    // it's ok to mutate the theme passed in because it's a clone
+    base.name = "tokyonight";
+    base.label = "Tokyo Night";
+    base.burst = true;
+    base.bg = "radial-gradient(circle, rgba(242,154,216,1) 5%, rgba(10,10,10,1) 70%)";

```

Comment:
I'm not quite sure what's gone wrong with this background but it doesn't seem to look anything like your screenshot. 

Something like this is a lot closer: 

```
    base.bg = "radial-gradient( circle, #69173e 5%, rgb(6 1 22) 100%)";
```